### ISSUE TITLE
Pagination buttons focus ring

### ISSUE THREAD

**Issue by itsam** [anubra266/choc-ui#34]
When using TAB to navigate the page, pagination buttons focus is not shown (the blue-ish ring around focused button).
That is a major accessibility issue and should be enabled (or maybe introduce a prop to define on demand)

Thank you in advance, great work!

**Comment by anubra266** [comment]
So I decided to go with prop. What propName do you suggest?

**Comment by anubra266** [comment]
So I decided to go with the `focusRing` prop
